ui/main/src/app/services/translation/TranslationService.ts

```
@@ -11,34 +11,40 @@
     public static setConfigServer(configServer: ConfigServer) {
         this.configServer = configServer;
     }
 
-    public static setTranslationService(translationService: TranslationService) {
-        this.translationService = translationService;
+    public static setTranslationLib(translationLib: TranslationLib) {
+        this.translationLib = translationLib;
+    }
+
+    public static setLang(lang: string): void {
+        return this.translationLib.setLang(lang);
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)